go/base/static/templates/apps/dialogue/states/httpjson/preview.jst

```diff
@@ -0,0 +1,6 @@
+<p class="text">
+Http Json Method: <span class="httpjson-method"><%= method %></span> 
```

Comment:
``Http Json Method`` -> ``HTTP JSON method`` and ``Http Json Url`` -> ``HTTP JSON URL``.